cypress/e2e/admin/adminRegistersTeacherWithLink.feature

```diff
@@ -1,37 +1,69 @@
-        Given I am logged in as '<user>' on '<env>'
-        And I am on the teachers management page
-        When Going to teacher edit page for 'John' with email 'john.teacher@mail.tld'
-        And Going to teacher registration page
-        And I am registering as teacher with email 'john.teacher@mail.tld' and password 'Password1!' on '<env>'
-        Then I am able to logout
+        # Admin creates a new teacher
+        When I click on administration in menu
+        When I go to '<role_to_manage>' administration
+        When I click on FAB
+        When I click on Add User in opened FAB for '<role_to_manage>'
+        When I fill out the user creation form for '<user_firstname>' '<user_lastname>' with email '<user_email>'
+        When I click on add button to add '<role_to_manage>'
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        Then I can see the user with email '<user_email>' in the table
 
-        Examples:
-            | env | user       |
-            | dbc | admin1_dbc |
+        # Admin generates the registeration link to proceed with teacher registration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        Then I click on the button Generate Personal Registration Link for teacher
+
+        # Teacher performs registration via the generated link with PIN
+        Then I visit to the generated student registration link
+        When I choose the language for the registration process
+        Then I click on the button Next to proceed to next step
+        Then I click again on the button Next to proceed to the personal data information page
+        Then I see my first name '<user_firstname>'
+        Then I see my last name '<user_lastname>'
+        Then I set a new password on personal data page
+        When I click on the button Next to proceed to the next step
+        When I accept the privacy and terms of use consents
+        Then I click on the button Next to proceed to the registration pin step
+        When I request a new registration pin
+        Then I retrieve the registration pin to enter it into the form for '<namespace>'
+        Then I click on the button Send and Get Started to successfully complete the registration process
+        Then I see the summary page
+
+        # Newly registered teacher does the first login
+        When I visit the url for first login
+        When I enter the email assigned during user creation
+        When I enter the set password
+        When I click on the button Login
+        When I click on the button Next on the section 1
+        Then I see my email
+        When I click on the button Next on the section 2
+        When I click on the button Get started right away on the section 3
+        Then I see the dashboard
+        Then I logout from the application
```

Comment:
I think we don't need that step

Reply:
In this tests its needed, also due to session, can you try in your local gui cy?

Reply:
done, removed from this features.